WcaOnRails/spec/mailers/competitions_mailer_spec.rb

```diff
@@ -186,16 +186,38 @@
       CompetitionsMailer.notify_of_delegate_report_submission(competition)
     end
 
-    it "renders the headers" do
-      expect(mail.subject).to eq "[wca-report] [Oceania] Comp of the Future 2016"
-      expect(mail.to).to eq ["reports@worldcubeassociation.org"]
-      expect(mail.cc).to match_array competition.delegates.pluck(:email)
-      expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
-      expect(mail.reply_to).to match_array competition.delegates.pluck(:email)
+    context "wrc feedback requested" do
+      before(:each) do
+        competition.delegate_report.update_attributes!(wrc_feedback_requested: true, wrc_incidents: "1, 2, 3")
+      end
+
+      it "renders the headers" do
+        expect(mail.subject).to eq "[wca-report] [Oceania] Comp of the Future 2016"
+        expect(mail.to).to eq ["reports@worldcubeassociation.org"]
+        expect(mail.cc).to match_array competition.delegates.pluck(:email) + ["reports@worldcubeassociation.org"]
```

Comment:
I think this is supposed to be `regulations@`, not `reports@`. See test failure [here](https://travis-ci.org/thewca/worldcubeassociation.org/builds/565285488#L4773-L4780)

Reply:
Oof. 
Just fixed! :)